CHANGELOG.md

```diff
@@ -0,0 +1,129 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic
+Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
+
+## [Unreleased]
+
+### Added
+
+- Add Coveralls support
+- Add testing with Tarantool 2.x-latest
+- Add reusable testing workflow (integration testing with latest Tarantool)
+- Add simple CI based on GitHub actions (#114)
+- Add support UUID type in msgpack (#90)
+- Add go modules support (#91)
+- Add utube handling (#85)
+
+### Fixed
+
+- Fix queue tests (#107)
+
+### Changed
+
+- Handle everything with go test (#115)
+- Use plain package instead of module (#134)
+- Make test case consistent with comments (#105)
+- Reset buffer if its average use size smaller than quater of capacity (#95)
+
+## [1.5] - 2019-12-29
+
+First release.
+
+### Fixed
+
+- Fix `Upsert()` for `ConnectionMulti` infinite recursive call
+- Fixed index out of range panic on `dial()` to short address
+- Fix cast in `defaultLogger.Report` (#49)
+- Fix race condition on extremely small request timeouts (#43)
+- Fix notify for `Connected` transition
+- Fix reconnection logic and add `Opts.SkipSchema()`
+- Fix future sending
+- Fix panic on disconnect + timeout
+- Fix block on msgpack error
+- Fix ratelimit
+- Fix timeouts
+- Fix mistake
+- Fix race condition (?)
```

Comment:
?

Reply:
Original commit message:

```
commit 6b262217ee353f0440358090e04ff9eeb1f8d162
Author: Sokolov Yura aka funny_falcon <funny.falcon@gmail.com>
Date:   Thu Jun 29 11:19:06 2017 +0300

    fix race condition on extremely small request timeouts (I hope)
    
    Fixes #43 (I hope)
```

Reply:
```diff
@@ -46,7 +46,7 @@ First release.
 - Fix ratelimit
 - Fix timeouts
 - Fix mistake
-- Fix race condition (?)
+- Fix race condition
 - Fix race condition on future channel creation
 - Fix block on forever closed connection
 - Fix race condition in Connection
```

Reply:
Then why not "fix race condition on extremely small request timeouts"

Reply:
```diff
@@ -46,7 +46,7 @@ First release.
 - Fix ratelimit
 - Fix timeouts
 - Fix mistake
-- Fix race condition
+- Fix possible race condition on extremely small request timeouts
 - Fix race condition on future channel creation
 - Fix block on forever closed connection
 - Fix race condition in Connection
```